### ISSUE TITLE
Import b9c1235d15: Revert "Add jump binding." (by Caity)

### ISSUE THREAD

**Issue by daleglass** [overte-org/tivolicloud#173]
This is an automated proposal to look at a commit made by Caity and import it into [Overte](https://github.com/overte-org/overte)

| **Commit:** |  b9c1235d15b0e35172e85a007f4d59e45274b90a |
| --- | --- |
| **Author:** | Caity |
| **Date:** | Thu, 06 Feb 2020 01:28 |

```
Revert "Add jump binding."

This reverts commit 006e2d09b3edb3f847aa068cdfbbc87971f3434b.

```

**Stats:**

| **Filename** | Stats | Lines | Added | Removed | Lines in blame |
| --- |  --- | --- | --- | --- | --- |
| interface/resources/controllers/standard.json |  | 40 | 26 | 14 | 26
| 1 files | - | 40 | 26 | 14 | 26 |

To work on this, please assign the issue to yourself, then look at the commit and decide whether this would be a good addition to Overte.

* If the commit is useful, tag it with "Tivoli: Keep", and keep it open until it's merged.
* If the commit is not useful, tag it with "Tivoli: Discard", and close it.
* If the commit is not useful right now, but might be later, tag it with "Tivoli: Maybe later", and close it.
* If it's hard to decide, tag it with "Tivoli: Discuss", and keep it open.

Useful commits should be submitted as a [PR against Overte](https://github.com/overte-org/overte/pulls). Tag this issue in PR, so that it's automatically closed once the PR is merged.

You can cherry-pick this issue with this command:

    git cherry-pick b9c1235d15b0e35172e85a007f4d59e45274b90a

**Comment by daleglass** [comment]
Duplicate of #133
